h2o-core/src/main/java/water/parser/BufferedString.java

```diff
@@ -150,17 +151,24 @@ public void setOff(int off) {
       for (int i = 0; i < _len; ++i)
         if (_buf[_off + i] != str._buf[str._off + i]) return false;
       return true;
-    } // FIXME: Called in NA_String detection during CsvParser, UTF-8 sensitive
```

Comment:
It was removed because it was the code that breaks the rules for equals() - symmetry. See "Effective Java". The comment was about that bad piece.